src/cirrus/build.py

```diff
@@ -45,6 +67,13 @@ def main():
         'venv',
         'bin',
         'virtualenv')
+
+    # remove existing virtual env if building clean
+    if opts.clean is True and os.path.exists(venv_path):
+        cmd = "rm -r {0}".format(venv_path)
```

Comment:
The `f` is only needed if you do not have permissions to change the file, which you should have in your local dev environment and I believe we'll have in the CI environment.